hw/ip/prim/dv/prim_prince/prim_prince_sim_cfg.hjson

```diff
@@ -42,6 +42,8 @@
     {
       name: smoke
       tests: ["prim_prince_test"]
+      run_opts: ["+smoke_test=1"]
+      reseed: 1
```

Comment:
In private CI, we set `fixed-seed=1`, which will only run one iteration. I think it should be okay to remove this line.

Reply:
Actually it is added here already: 
https://github.com/lowRISC/opentitan/blob/66a434d868e2f82164c3321056098ce9374ffbaa/hw/dv/tools/dvsim/common_sim_cfg.hjson#L113

Reply:
I did not know there is a master config here! Maybe we can put the run option `smoke_test` here instead?

Reply:
hm agree with Cindy, could we move the `smoke_test` runopt to this master config to centralize it?

Reply:
Sure!

Reply:
Sounds good then! I will add it to PR #4947 !

Reply:
Thanks Cindy! I will remove the plusarg from this PR and will merge once  #4947 goes in.

Reply:
like the idea of centralize this knob. perhaps we can get it in dv_base_vseq as well and limit the `num_trans` to just 1 or 2 if it's a smoke test

Reply:
Yeah I hesitated to do that. But definitely could add it. The reason i did not was because - most smoke_test_vseq ended up override the `num_trans_c` constraints. So was not very affectively. But I am neutral to it  :)